From ef1781dac2ae30a76fd4d2f156b16e3262c0ec74 Mon Sep 17 00:00:00 2001
From: chris41g <chriswhite41g@gmail.com>
Date: Mon, 6 Aug 2012 16:59:19 -0500
Subject: [PATCH 24/24] derp

---
 include/linux/sched.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/linux/sched.h b/include/linux/sched.h
index 8b84dfd..9a6a34d 100644
--- a/include/linux/sched.h
+++ b/include/linux/sched.h
@@ -122,7 +122,7 @@ extern void get_avenrun(unsigned long *loads, unsigned long offset, int shift);
 
 #define FSHIFT		11		/* nr of bits of precision */
 #define FIXED_1		(1<<FSHIFT)	/* 1.0 as fixed-point */
-#define LOAD_FREQ	((4*HZ+61)  /* 4.61 sec intervals */
+#define LOAD_FREQ	((4*HZ+61))  /* 4.61 sec intervals */
 #define EXP_1		1884		/* 1/exp(5sec/1min) as fixed-point */
 #define EXP_5		2014		/* 1/exp(5sec/5min) */
 #define EXP_15		2037		/* 1/exp(5sec/15min) */
-- 
1.7.5.4

